src/mon/OSDMonitor.cc

```diff
@@ -1128,30 +1129,40 @@ void OSDMonitor::encode_pending(MonitorDBStore::TransactionRef t)
 
   // tell me about it
   for (auto i = pending_inc.new_state.begin();
-       i != pending_inc.new_state.end();
-       ++i) {
-    int s = i->second ? i->second : CEPH_OSD_UP;
+       i != pending_inc.new_state.end(); ++i) {
+    int32_t s = i->second ? i->second : CEPH_OSD_UP;
+    int osd = i->first;
     if (s & CEPH_OSD_UP)
-      dout(2) << " osd." << i->first << " DOWN" << dendl;
+      dout(2) << __func__ << " "
+              << "osd." << osd << " DOWN" << dendl;
     if (s & CEPH_OSD_EXISTS)
-      dout(2) << " osd." << i->first << " DNE" << dendl;
-  }
-  for (map<int32_t,entity_addr_t>::iterator i = pending_inc.new_up_client.begin();
-       i != pending_inc.new_up_client.end();
-       ++i) {
-    //FIXME: insert cluster addresses too
-    dout(2) << " osd." << i->first << " UP " << i->second << dendl;
-  }
-  for (map<int32_t,uint32_t>::iterator i = pending_inc.new_weight.begin();
-       i != pending_inc.new_weight.end();
-       ++i) {
-    if (i->second == CEPH_OSD_OUT) {
-      dout(2) << " osd." << i->first << " OUT" << dendl;
-    } else if (i->second == CEPH_OSD_IN) {
-      dout(2) << " osd." << i->first << " IN" << dendl;
-    } else {
-      dout(2) << " osd." << i->first << " WEIGHT " << hex << i->second << dec << dendl;
-    }
+      dout(2) << __func__ << " "
+              << "osd." << osd << " DNE" << dendl;
+  }
+  for (std::map<int32_t, entity_addr_t>::iterator i =
```

Comment:
@shinobu-x instead of mixing the changes of reformatting the code with those implementing the new features, i think it would be better to split the change into smaller commits to help the reviewer understand your intention.

also, i am not sure why you are adding `std::` here?

Reply:
@tchaikov 
> also, i am not sure why you are adding std:: here?

Simply I don't like using global namespace. I know this is kind of religious wars...
Anyway I will split this patch into several pieces.